### ISSUE TITLE
Font is not displayed on win-arm64 from 11.0.0-preview5 and onward

### ISSUE THREAD

**Issue by davte-beijer** [AvaloniaUI/Avalonia#13206]
## Describe the bug
When building Avalonia projects targeting win-arm64 from 11.0.0-preview5 and onward no font will be displayed in the application. 

## To Reproduce
1. Follow the getting started guide to build Avalonia
2. From repository root run `dotnet publish samples\ControlCatalog.NetCore\ControlCatalog.NetCore.csproj -c Release --self-contained true -r win-arm64`
3. Run the application on an win-arm64 device

## Expected behavior
Font should be visible

## Screenshots
11.0.0-preview5 and onward:
![image](https://github.com/AvaloniaUI/Avalonia/assets/25159662/f2fe0906-9af9-4089-8fb8-f4ce511884cf)

11.0.0-preview4:
![image](https://github.com/AvaloniaUI/Avalonia/assets/25159662/b667943b-974a-411b-8487-ad8ba71b371b)

## Environment 

- Arch: arm64 (win-arm64)
- OS: Windows 10 IoT Enterprise
- Avalonia-Version: 11.0.4

**Comment by timunie** [comment]
Is the font available on the system?

**Comment by davte-beijer** [comment]
Yes, I have installed the Inter Font separately on the target, but makes no difference. When I run projects on my win-x64 machine it works fine and I do not have the Inter font installed.

**Comment by maxkatz6** [comment]
Can you try to downgrade Angle version used in the project? By referencing this version of the angle package: https://www.nuget.org/packages/Avalonia.Angle.Windows.Natives/2.1.0.2023020321:
```xml
<PackageReference Include="Avalonia.Angle.Windows.Natives" Version="2.1.0.2023020321" />
```

Also, it seems not all windows on arm devices have old known compatibility issue - https://github.com/AvaloniaUI/Avalonia/issues/10405. Unless you are using software rendering from the beginning.

**Comment by maxkatz6** [comment]
Upd: just remembered that 11.0.5-rc should use the older Angle package anyway. No idea what could go wrong then.

**Comment by davte-beijer** [comment]
Please update me if you need any other scenario verified!

**Comment by timunie** [comment]
Out of curiosity: Do you see any logs that could give us a hint?

**Comment by timunie** [comment]
I also wonder if we have a chance to find the commit that broke it for you.

**Comment by davte-beijer** [comment]
I made a smaller project based on the mvvm template and configured logging like the following:
```csharp
 public static AppBuilder BuildAvaloniaApp()
        => AppBuilder.Configure<App>()
            .UsePlatformDetect()
            .WithInterFont()
            .LogToTrace(LogEventLevel.Verbose)
            .UseReactiveUI();
```

The logs do not mention anything that seem to be font issues, at least from what I can tell.

[win-arm64.log](https://github.com/AvaloniaUI/Avalonia/files/12886715/win-arm64.log)
[win-x64.log](https://github.com/AvaloniaUI/Avalonia/files/12886716/win-x64.log)
